cmd/image-builder/version.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"runtime/debug"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+)
+
+var (
+	version string
+	commit  string
+
+	imagesVersion string
+)
+
+func init() {
```

Comment:
Not sure this is needed in init() ? Given that its used in prettyVersion() this could be a normal helper or am I missing something?

Reply:
There was no specific reason, my thinking was if this is a "constant" string, then I would set it in `init` once and never touch it. Returning a copy via a helper is probably safer now when I think about it.

Reply:
I've adjusted `init` to be a helper instead which returns a `*versionDescription` which is then used directly by `prettyVersion`.